최창익/app/src/main/java/com/mash_up/mvvmstudy/view/detail/DetailActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.mash_up.mvvmstudy.view.detail
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.mash_up.mvvmstudy.databinding.ActivityDetailBinding
+import com.mash_up.mvvmstudy.databinding.ActivityMainBinding
+
+class DetailActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityDetailBinding
+    private val viewModel: DetailViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityDetailBinding.inflate(layoutInflater)
+        setContentView(binding.root)
```

Comment:
```suggestion
binding = DatabindingUtil.inflate(this, R.layout.activity_main)
```
데이터 바인딩을 이용중이시라면 한줄로 줄일 수 있을 것 같아요!

Reply:
오 좋습니다 반영할게요~!